source/js/foundation/template-js-handler/share-button-group.js

```diff
@@ -0,0 +1,45 @@
+import { ReactGA } from "../../common";
+import copyToClipboard from "../../copy-to-clipboard";
+
+export default () => {
+  const shareButtonGroups = document.querySelectorAll(
+    ".share-button-group-wrapper"
+  );
+
+  if (!shareButtonGroups) {
+    return;
+  }
+
+  let bindGaEvent = (type = ``) => {
+    ReactGA.event({
+      category: `social`,
+      action: `${type} share tap`,
+      label: `${type} share`,
+      transport: `beacon`,
+    });
+  };
+
+  let linkButtonClick = (event) => {
+    event.preventDefault();
+    const linkButton = event.target;
+
+    linkButton.classList.add("copied");
+    linkButton.title = window.gettext("Copied");
+    linkButton.querySelector("span").innerText = window.gettext("Copied");
+
+    copyToClipboard(event.target, window.location.href);
+    bindGaEvent("link");
+  };
+
+  shareButtonGroups.forEach((shareButtonGroup) => {
+    const facebookButton = shareButtonGroup.querySelector(".facebook-share");
+    const twitterButton = shareButtonGroup.querySelector(".facebook-share");
```

Comment:
this is definitely not the right query selector =D

Reply:
Wow! thank you for that catch Pomax! I have updated it and re-requested review